operator/manifests/kiali-upstream/kiali.package.yaml

```diff
@@ -1,7 +1,7 @@
 packageName: kiali
 channels:
 - name: alpha
-  currentCSV: kiali-operator.v1.10.0
+  currentCSV: kiali-operator.v1.11.0
 - name: stable
-  currentCSV: kiali-operator.v1.10.0
+  currentCSV: kiali-operator.v1.11.0
```

Comment:
This is the version I'm referring to - we should move these to 1.12.0. The names of the CSV yamls need to be renamed too (e.g. kiali.v1.12.0.clusterserviceversion.yaml)

Reply:
fyi: I'm going to create a 1.11 set of manifests when we release this sprint